functions/ai/resume_parser.py

```
@@ -0,0 +1,87 @@
+    Args:
+        resume_file: A FileStorage object representing the uploaded resume.
+
+    Returns:
+        A dictionary containing the structured user profile data.
+    """
+    try:
+        # Step 1: Extract text from the resume file
+        resume_text = _extract_text_from_file(resume_file)
+
+        # Step 2: Construct prompt for Gemini to parse the resume
+        prompt = RESUME_PARSING_PROMPT_TEMPLATE.format(resume_text=resume_text)
```

Ruff (0.12.2)

Undefined name `RESUME_PARSING_PROMPT_TEMPLATE`

(F821)